javascript-loops/main.js

```diff
@@ -0,0 +1,93 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+function getNumbersToTen() {
+  const numbers = [];
+  let currentNumber = 1;
+
+  while (currentNumber <= 10) {
+    numbers.push(currentNumber);
+    currentNumber++;
+  }
+
+  return numbers;
+}
+
+function getEvenNumbersToTwenty() {
+  const evenNumbers = [];
+  let currentNumber = 2;
+
+  while (currentNumber <= 20) {
+    evenNumbers.push(currentNumber);
+    currentNumber += 1;
+  }
+
+  return evenNumbers;
+}
+
+console.log(getEvenNumbersToTwenty());
+console.log(getEvenNumbersToTwenty());
+
+function repeatWord(word, times) {
+  let count = 1;
+  let repeated = '';
+
+  while (count <= times) {
+    repeated += word;
+    count += 1;
+  }
+
+  return repeated;
+}
+
+console.log(repeatWord('Jack', 5));
+console.log(repeatWord('Jack', 4));
+console.log(repeatWord('Jack', 3));
+console.log(repeatWord('Jack', 2));
+console.log(repeatWord('Jack', 1));
+
+function logEachCharacter(string) {
+  for (let i = 0; i < string.length; i += 1) {
+    console.log(string);
+  }
+}
+
+console.log('weokooio');
+
+function doubleAll(numbers) {
+  const doubled = [];
+  for (let i = 0; i < numbers.length; i += 1) {
+    const doubledTwo = numbers.map((arr) => arr * 2);
+    doubled.push('doubleAll:', doubledTwo);
```

Comment:
This is technically pushing 2 values into the array. You only want to push 1 value. This isn't a `console.log` where we need to include a label, so we don't need the string `'doubleAll:`